installation/scripts/testing-common.sh

```diff
@@ -51,7 +69,8 @@ function printLogsFromFailedTests() {
         "Pending")
             log "'${pod}' failed due to too long Pending status" red
             printf "Fetching events from '${pod}':\n"
-            kubectl $(context_arg)  describe po ${pod} -n ${namespace} | awk 'x==1 {print} /Events:/ {x=1}'
+            executeKubectlWithRetries "kubectl $(context_arg)  describe po ${pod} -n ${namespace} | awk 'x==1 {print} /Events:/ {x=1}'"
+            echo $result
```

Comment:
why print that?

Reply:
i print that to log kubectl describe result